packages/react/src/components/numericInput/NumericInputConnected.tsx

```diff
@@ -119,6 +119,7 @@ export class NumericInputConnected extends PureComponent<NumericInputProps & HTM
             <div className="numeric-input flex flex-column">
                 <div className="flex flex-row">
                     <button
+                        type="button"
```

Comment:
Do we want enzyme tests to ensure this prop is on the buttons? I feel it's pretty overkill however it could document why these types are important

Reply:
I could make them in testing library also if it's preferable and the setup is not too daunting